### PR TITLE
02. GIT. Local task

### PR DESCRIPTION
**Pull request by YuliyaHancharenka**

### CODE REVIEW

File: Yuliya_Hancharenka/02.Git.Local/commit_feature.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Commit to feature branch
```
Comment (pluhin) [inline Yuliya_Hancharenka/02.Git.Local/commit_feature.txt]: I need just history, could you please remove these txt files

File: Yuliya_Hancharenka/02.Git.Local/02.GIT.Local.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+## Git log history
+
```
Comment (pluhin) [inline Yuliya_Hancharenka/02.Git.Local/02.GIT.Local.md]: here should be tag ```

**Comment by pluhin** [comment]
Could you please provide list of commits each branches finally

File: Yuliya_Hancharenka/02.Git.Local/commit_feature.txt (hunk 1 above)
Comment (YuliyaHancharenka) [reply Yuliya_Hancharenka/02.Git.Local/commit_feature.txt]: done

File: Yuliya_Hancharenka/02.Git.Local/02.GIT.Local.md (hunk 2 above)
Comment (YuliyaHancharenka) [reply Yuliya_Hancharenka/02.Git.Local/02.GIT.Local.md]: done

**Pull request merged**